### PR TITLE
Fix linting error

### PR DESCRIPTION
**Pull request by Khaaz**
This PR fix the lint error introduced in 2fdf6c02b89a17ffef6532682f1e42bcc3455fc9 (#642).
Apparently I missed it and it was not catched by CI.

### CODE REVIEW

**Review by abalabahaha — DISMISSED** [review]

**Comment by Khaaz** [comment]
Edited this to also fix other linting error.
This makes this PR entirely fix linting for the current dev branch.

File: lib/gateway/Shard.js [hunk 1]
```diff
@@ -548,7 +548,7 @@ class Shard extends EventEmitter {
                 */
                 this.emit("messageReactionAdd", message || {
                     id: packet.d.message_id,
-                    channel: channel || { id: packet.d.channel_id }
+                    channel: channel || {id: packet.d.channel_id}
```
Comment (Skillz4Killz) [inline lib/gateway/Shard.js:552]: This needs typings changes. I saw it in another PR but forgot to comment on that that this breaks quite a bit of typings

File: lib/gateway/Shard.js [hunk 2]
```diff
@@ -578,7 +578,7 @@ class Shard extends EventEmitter {
                 */
                 this.emit("messageReactionRemove", message || {
                     id: packet.d.message_id,
-                    channel: channel || { id: packet.d.channel_id }
+                    channel: channel || {id: packet.d.channel_id}
```
Comment (Skillz4Killz) [inline lib/gateway/Shard.js:582]: Same here typings

File: lib/gateway/Shard.js (hunk 1 above)
Comment (Khaaz) [reply lib/gateway/Shard.js:552]: Could you fix that in another PR as this one is purely styling?

Comment (Khaaz) [reply lib/gateway/Shard.js:552]: for reference, introduced in #672

**Comment by Khaaz** [comment]
This fix all linting errors to the current dev branch state

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**